lib/auth/jwtclient.js

```diff
@@ -189,28 +187,22 @@ JWT.prototype.fromStream = function(stream, opt_callback) {
 };
 
 /**
- * Creates the GAPI instance if it has not been created already.
+ * Creates the gToken instance if it has not been created already.
  * @param {function=} callback Callback.
  * @private
  */
-JWT.prototype._createGAPI = function(callback) {
-  var that = this;
-
-  if (that.gapi) {
-    callback(null, that.gapi);
+JWT.prototype._createGToken = function(callback) {
+  if (this.gtoken) {
+    callback(null, this.gtoken);
   } else {
-    that.gapi = new that.GAPI({
-      iss: that.email,
-      sub: that.subject,
-      scope: that.scopes instanceof Array ? that.scopes.join(' ') : that.scopes,
-      keyFile: that.keyFile,
-      key: that.key
-    }, function (err) {
-      if (err) {
-        that.gapi = null;
-      }
-      callback(err, that.gapi);
+    this.gtoken = this.gToken({
+      iss: this.email,
+      sub: this.subject,
+      scope: this.scopes instanceof Array ? this.scopes.join(' ') : this.scopes,
```

Comment:
gtoken will accept both array of scopes or space delimited string so `scope: this.scopes` should be sufficient.